tools/release-note.md

```diff
@@ -1,21 +1,11 @@
-This is release `${CIRCLE_TAG}` of Loki. 
+This is release `${CIRCLE_TAG}` of Loki.
 
 ### Notable changes:
 :warning: **ADD RELEASE NOTES HERE** :warning:
 
 ### Installation:
 The components of Loki are currently distributed in plain binary form and as Docker container images. Choose what fits your use-case best.
 
-#### Binary:
-```bash
-# download a binary (adapt app, os and arch as needed)
-$ curl -fSL -o "/usr/local/bin/loki.gz" "https://github.com/grafana/loki/releases/download/${CIRCLE_TAG}/loki-linux-amd64.gz"
-$ gunzip "/usr/local/bin/loki.gz"
-
-# make sure it is executable
-$ chmod a+x "/usr/local/bin/loki"
-```
-
```

Comment:
I think removing this entirely is a little bit too much .. We still need to let users know where to get the binaries from.

What about:

```md
#### Binary
We provide pre-compiled binary executables for the most common operating systems and architectures.
Pick one from the list below, download it (e.g. using `curl`) and place it somewhere in your `$PATH`.
Do not forget to mark it as executable (`chmod +x`).
```

Reply:
done.